monolith/templates/new_story.html

```diff
@@ -0,0 +1,19 @@
+<html>
+
+<body>
+  <h1>Create a new story</h1>
+  <form action="/write_story" method="POST">
```

Comment:
Following the comment on /new_story [GET] this should send a post request to /write_story [POST] and there we create the empty unpublished story in the db.